tests/filecheck/dialects/arm_neon/test_ops.mlir

```diff
@@ -0,0 +1,9 @@
+// RUN: XDSL_ROUNDTRIP
+// RUN: xdsl-opt -t arm-asm %s | filecheck %s --check-prefix=CHECK-ASM
+// CHECK: %v1 = arm_neon.get_register : !arm_neon.reg<v1>
+%v1 = arm_neon.get_register : !arm_neon.reg<v1>
+// CHECK: %v2 = arm_neon.get_register : !arm_neon.reg<v2>
+%v2 = arm_neon.get_register : !arm_neon.reg<v2>
+// CHECK: %dss_fmulvec = arm_neon.dss.fmulvec %v1, %v2 {arrangement = #arm_neon<arrangement S>, comment = "floating-point vector multiply v1 by v2", scalar_idx = 0 : i8} : (!arm_neon.reg<v1>, !arm_neon.reg<v2>) -> !arm_neon.reg<v3>
+// CHECK-ASM: fmul v3.4S, v1.4S, v2.S[0] # floating-point vector multiply v1 by v2
+%dss_fmulvec = arm_neon.dss.fmulvec %v1, %v2 {"arrangement" = #arm_neon<arrangement S>, "comment" = "floating-point vector multiply v1 by v2", "scalar_idx" = 0 : i8} : (!arm_neon.reg<v1>, !arm_neon.reg<v2>) -> !arm_neon.reg<v3>
```

Comment:
Would something like this be possible? feels like it would be nice to have this crucial info a bit more prominent in the custom syntax (`arrangement S` instead of just `S` would also work if that's easier`

```suggestion
%dss_fmulvec = arm_neon.dss.fmulvec S %v1, %v2[0] {"comment" = "floating-point vector multiply v1 by v2",} : (!arm_neon.reg<v1>, !arm_neon.reg<v2>) -> !arm_neon.reg<v3>
```

Reply:
sure, please see next commit